node/pkg/governor/governor_test.go

```diff
@@ -1936,3 +2186,84 @@ func TestPendingTransferWithBadPayloadGetsDroppedNotReleased(t *testing.T) {
 	_, exists = gov.msgsSeen[gov.HashFromMsg(&msg2)]
 	assert.False(t, exists)
 }
+
+func TestCheckedAddUint64HappyPath(t *testing.T) {
```

Comment:
We can come back later and replace these with [table driven tests](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests) but these are nice as is.